README.md

```diff
@@ -66,12 +61,18 @@ An introductory blog post was
 [published on Medium][medium-pubgrub] by its author.
 
 The detailed explanation of the algorithm is
-[provided on GitHub][github-pubgrub].
+[provided on GitHub][github-pubgrub],
+and complemented by the ["Internals" section of our guide][guide-internals].
 The foundation of the algorithm is based on ASP (Answer Set Programming),
 and a book called
 "[Answer Set Solving in Practice][potassco-book]"
 by Martin Gebser, Roland Kaminski, Benjamin Kaufmann and Torsten Schaub.
 
+[crates]: https://crates.io/crates/pubgrub
+[guide]: https://pubgrub-rs-guide.netlify.app/
+[guide-internals]: https://pubgrub-rs-guide.netlify.app/internals/intro.html
```

Comment:
Should we use `guide-dev` for consistency?

Reply:
this link is specifically for internals, not for a dev version of the guide